tinymce/views.py

```diff
@@ -4,6 +4,12 @@
 import json
 import logging
 
+import django
+if django.VERSION < (2,):  # pragma: no cover
+    import warnings
+    warnings.warn("Support for Django < 2.0 will be removed soon,"
+                  "please upgrade your projects to use Django 2.0", DeprecationWarning)
+
```

Comment:
I guess we don't need https://github.com/aljosa/django-tinymce/pull/224/files#diff-26390c0f125e430f2d444a575e6004baR18 anymore

Reply:
You mean this piece of code:
```
 try:
     from django.urls import reverse
 except ImportError:
     # Django < 1.10
from django.core.urlresolvers import reverse
```


I think we still need it because we've not yet dropped support for Django 1.11 or anything before that. That should be removed once the warning above becomes a `raise DeprecationWarning`.

Reply:
@Natim ping, please see last comment and let me know if there's still something left to do for this PR.

Reply:
Since we don't break compatibility yet it is better to warn user that support will stop eventually.
With the slow pace updates of the lib I feel like now is a good time. Plus starting to upgrade to Django 2,0 doesn't seems like a bad idea for project to start considering.